lib/Net/Google/Drive/Simple.pm

```diff
@@ -215,9 +217,10 @@ sub file_upload {
         $url = URI->new( $self->{ api_file_url } );
 
         my $data = $self->http_json( $url,
-            { mimeType => $mime_type,
-              parents  => [ { id => $parent_id } ],
-              title    => $title,
+            { mimeType    => $mime_type,
+              parents     => [ { id => $parent_id } ],
+              title       => $opts->{ title } ? $opts->{ title } : $title,
+              description => $opts->{ description },
```

Comment:
description should be optional too
`defined $opts->{ description } ? ( description => $opts->{ description } ) : ()`